programs/clearing_house/src/controller/spot_balance.rs

```diff
@@ -39,34 +39,57 @@ pub fn update_spot_market_twap_stats(
         spot_market,
         &SpotBalanceType::Deposit,
     )?;
+    msg!("deposit_token_amount:{}", deposit_token_amount);
```

Comment:
lotta extra msg statements throughout this pr, can you go back through all the files and remove em?